src/bindings/python/src/openvino/frontend/tensorflow/utils.py

```diff
@@ -369,7 +369,7 @@ def extract_model_graph(argv):
     if isinstance(model, tf.compat.v1.Session):
         argv["input_model"] = model.graph
         return True
-    if env_setup["tensorflow"] >= LooseVersion("2.6.0") and isinstance(model, (tf.types.experimental.GenericFunction,
+    if Version(env_setup["tensorflow"]) >= parse("2.6.0") and isinstance(model, (tf.types.experimental.GenericFunction,
```

Comment:
@popovaan is it mandatory to have this version check? is it possible to detect proper case somehow else? e.g. analyze object attributes, etc

Reply:
We can check version using `tensorflow.__version__` attribute. But it will be a `str`, so we need to make a method to compare which version is more or less.

Reply:
The thing that worries me to create a method which compares versions instead of using packaging that versions may be more complicated than three digits with dots. And we will have to manually support [each corner case](https://semver.org/).

Reply:
@ilya-lavrenov I am also against creating custom checks for things that are already implemented. The `packaging` module is under the well-known Python Packaging Authority and addresses the issues that we are actually trying to resolve. In addition, the package itself is pure Python and around tens of kb -- small cost in comparison to writing custom implementation of all checks/tooling.

What are the reasons to be against this tool? Do you see (or predict) any critical events that may happen from replacing native Python module with one that still follows PEP rules?